core/templates/dev/head/pages/topic-editor-page/services/topic-editor-state.service.spec.ts

```diff
@@ -268,6 +271,11 @@ describe('Topic editor state service', function() {
         skill_2: 'Description 2',
         skill_3: 'Description 3',
         skill_5: 'Description 5'
+      },
+      skillIdToRubricsDict: {
```

Comment:
Add some case where the rubric isn't an empty object

Reply:
Done